### PR TITLE
Release 1.0.3

### PR DESCRIPTION
**Pull request by mnecas**
@mwperina @jmolmo @pcuzner

### CODE REVIEW

File: misc/packaging/ansible-runner-service.spec [hunk 1]
```diff
@@ -75,6 +75,10 @@ install -m 0644 ./LICENSE.md %{buildroot}%{_docdir}/ansible-runner-service
 %{_docdir}/ansible-runner-service/*
 
 %changelog
+* Thu Jun 4 2020 Martin Necas <mnecas@redhat.com> 1.0.3-1
+- Add logrotate
```
Comment (mwperina) [inline misc/packaging/ansible-runner-service.spec:79]: We haven't changed anything there, right? If so, we should mention that there was no change

File: misc/packaging/apache/ansible-runner-service.spec [hunk 2]
```diff
@@ -74,6 +74,10 @@ install -m 644 ./ansible_runner_service.py %{buildroot}%{python3_sitelib}/runner
 %doc README.md
 
 %changelog
+* Thu Jun 4 2020 Martin Necas <mnecas@redhat.com> 1.0.3-1
+- Add logrotate
```
Comment (mwperina) [inline misc/packaging/apache/ansible-runner-service.spec:78]: Add logrotate configuration to purge old log files

File: misc/packaging/nginx/ansible-runner-service-nginx.spec [hunk 3]
```diff
@@ -93,6 +93,10 @@ install -m 0644 ./misc/nginx/uwsgi.ini %{buildroot}%{_sysconfdir}/ansible-runner
 
 
 %changelog
+* Thu Jun 4 2020 Martin Necas <mnecas@redhat.com> 1.0.3-1
+- Add logrotate
+- Add psutil to dependencies
```
Comment (mwperina) [inline misc/packaging/nginx/ansible-runner-service-nginx.spec:98]: We haven't change anything there, right? If so, we should mention that

File: packaging/gunicorn/ansible-runner-service.spec [hunk 4]
```diff
@@ -83,6 +89,10 @@ cp -r ./packaging/gunicorn/ansible-runner-service.service %{buildroot}%{_unitdir
 %doc README.md
 
 %changelog
+* Thu Jun 4 2020 Martin Necas <mnecas@redhat.com> 1.0.3-1
+- Add logrotate
```
Comment (mwperina) [inline packaging/gunicorn/ansible-runner-service.spec:93]: Add logrotate configuration to purge old log files

**Comment by jmolmo** [comment]
It seems ok for me. Just one question:
Have you tried to build the nginx container with the new "psutil" dependency and check that it starts?

**Comment by mwperina** [comment]
AFAIK the psutil dependency raised up only when tried to use ARS RPM under Apache on CentOS Stream, but we didn't saw the issue on standard CentOS 8.1. That's why I we haven't touched spec file for nginx container, am I right Martine?

Anyway let's check also nginx container build

@mnecas

**Comment by mnecas** [comment]
Yes, that is correct. 
@jmolmo if you want I can add it to nginx just to be safe.

**Comment by jmolmo** [comment]
> Yes, that is correct.
> @jmolmo if you want I can add it to nginx just to be safe.

Yes, it will be nice. Probably is safer. Once "psutil" dependency will be included i think that it is ok to create the new release.
Thank you

**Pull request merged**